package.json

```diff
@@ -2,29 +2,31 @@
   "name": "@veupathdb/eda",
   "version": "0.14.11",
   "dependencies": {
-    "@emotion/babel-preset-css-prop": "^11.2.0",
     "@emotion/react": "^11.4.1",
     "@emotion/styled": "^11.3.0",
     "@material-ui/core": "^4.11.3",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.58",
     "@mui/styles": "^5.0.1",
     "@types/debounce-promise": "^3.1.3",
-    "@veupathdb/components": "^0.8.8",
     "@veupathdb/core-components": "^0.2.43",
+    "@veupathdb/web-common": "^0.1.10",
```

Comment:
This should not be a dependency. It is only needed as a devDependency, and will create a circular dependency if left here.
 
```suggestion
```